index.css

```diff
@@ -0,0 +1,739 @@
+  background-size: 100% auto;
+  background-color: #fff;
+  background-position: top 110px right 5%;
+  border-radius: 0 0 0 12.5rem;
+}
+
+.nav-wrapper {
+  position: fixed;
+  width: 100%;
+}
+
+nav {
+  padding: 26px 8%;
+  background-color: #fff;
+}
+
+a {
+  text-decoration: none;
+}
+
+.navigation {
+  display: flex;
+  padding: 0;
+}
+
+.navigation > li > a {
+  margin: 0 2rem;
+  font-size: 15px;
+  color: #344563;
+}
+
+nav > div > img:last-child {
+  display: none;
+  width: 35px;
+  height: 25px;
+  cursor: pointer;
+}
+
+.card > div > p {
+  color: #344563;
+  font-size: 16px;
+  margin-bottom: 28px;
+}
+
+.card > div:nth-child(2) {
+  padding: 24px 46px;
+}
+
+#logo {
+  font-family: 'Pacifico', cursive;
+  word-spacing: 0;
+  font-weight: bolder;
+  font-size: 1.5rem;
+  text-decoration: none;
+  font-style: oblique;
+  color: #6070ff;
+}
+
+.contact-container > h1 {
+  text-align: center;
+  margin: 0;
+}
+
+.personal {
+  padding-right: 25px;
+}
+
+.personal h3 {
+  font-size: 40px;
+}
+
+/* .personal p {
+  font-size: 15px;
+} */
+
+.headline {
+  margin: 0 17%;
+  padding: 20% 0;
+}
+
+.headline .title > h1 {
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.headline > div > p {
+  font-size: 20px;
+  color: #344563;
+  line-height: 28px;
+  font-weight: 400px;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(2, minmax(0, 1fr));
+}
+
+.space-y-3 > section:nth-child(2n + 2) .card > div:nth-child(2) {
+  grid-column-start: 1;
+  grid-row-start: 1;
+  padding: 24px 0;
+}
+
+.card {
+  background-color: #fff;
+  padding: 24px;
+  border-radius: 24px;
+  margin: 0 8%;
+}
+
+.p-highlight {
+  list-style: none;
+  padding-inline-start: 0;
+  margin-top: 0;
+}
+
+.p-highlight li::before {
+  content: '\2022';
+  color: #bcc0c9;
+  font-weight: bold;
+  font-size: 1.5rem;
+  display: inline-flex;
+  width: 1rem;
+  margin-left: -1rem;
+  margin-right: 0.3rem;
+  vertical-align: sub;
+}
+
+.p-highlight li:first-child::before {
+  color: transparent;
+  margin-right: 0;
+  text-transform: uppercase;
+}
+
+.p-highlight li {
+  font-size: 18px;
+  color: #7a869a;
+}
+
+.skill-set {
+  padding-left: 24px;
+}
+
+.skill-set > ul > li > img {
+  width: 48px;
+  height: 48px;
+}
+
+.skill-set > ul > li > span {
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+}
+
+.skill-set > ul {
+  display: flex;
+  margin: 0 0 28px;
+  padding: 12px;
+}
+
+.skill-set > ul > li {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  height: 120px;
+  width: 122px;
+  gap: 16px;
+  margin: 6px 12px;
+  padding: 12px;
+  border-radius: 8px;
+  color: #253858;
+  background-color: #f7f7f9;
+}
+
+.p-highlight li:first-child {
+  text-transform: uppercase;
+  font-weight: 700;
+  color: #344563;
+}
+
+.list-tags > * {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  color: #6070ff;
+  font-size: 12px;
+  border-radius: 10px;
+}
```

Comment:
- [ ] In your projects, kindly decrease the gap between the languages and the button. Also, change the title color to the right one as in the figma design:
your project:
>![image](https://user-images.githubusercontent.com/91486791/222826932-8a00b6a8-cd74-4de0-9da0-aabfb7e5545c.png)
figma:
>![image](https://user-images.githubusercontent.com/91486791/222826992-c1d759b8-1d91-41a7-aeb4-4d58bfc94dd5.png)